repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/nihub.ftl

```diff
@@ -0,0 +1,9 @@
+<#ftl output_format="HTML">
+<#include "../include/imports.ftl">
+<@hst.setBundle basename="nationalindicatorlibrary.headers"/>
```

Comment:
This bundle is unused

Reply:
+1 I guess it will be in next iteration but removing for now.

Reply:
Yeah that's fine you can leave it if you want, just making sure I wasn't missing something 🙂